packages/pds/src/db/migrations/20230804T224916736Z-action-duration.ts

```diff
@@ -0,0 +1,27 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await Promise.all([
+    db.schema
+      .alterTable('moderation_action')
+      .addColumn('actionDurationInHours', 'integer')
+      .execute(),
+    db.schema
+      .alterTable('moderation_action')
+      .addColumn('actionExpiresAt', 'varchar')
+      .execute(),
+  ])
```

Comment:
We tend to run schema updates serially, just to keep things simpler and deterministic.  The amount of time the migration takes to run isn't much of a concern.